src/main/resources/templates/show-student-result.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html xmlns:th="http://thymeleaf.org" lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+<body>
+<ul class="list-group" th:object="${student}" id="student">
```

Comment:
It seems, this page similar to create-result.
Could we use generic one for both cases?

Reply:
Made generic page.

Reply:
Don't see any real changes)
I mean, that we should use one page for displaying and another one might not be needed